src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php

```diff
@@ -431,7 +432,7 @@ protected function getDoctrine()
     /**
      * Get a user from the Security Token Storage.
      *
-     * @return object|null
+     * @return UserInterface|object|null Can be a UserInterface instance, an object implementing a __toString method, or the username as a regular string
```

Comment:
Just realised that a string will never be returned because of the `is_object()` check.